galadriel/cli.py

```diff
@@ -277,6 +278,24 @@ def import_wallet(private_key: str, path: str):
         click.echo(f"Successfully imported Solana wallet from {path}")
 
 
+@wallet.command()
+def airdrop():
+    """Request an airdrop of 0.0001 SOL to the given Solana wallet."""
+    key_path = os.getenv("SOLANA_KEY_PATH")
+    if not key_path or not os.path.exists(key_path):
+        raise click.ClickException("SOLANA_KEY_PATH not found in environment or does not exist")
```

Comment:
If someone uses it while not having a wallet I'd bet they want to have a wallet. Maybe worth creating one instead of raising the execption?

Reply:
It could also be the case that user has already created the wallet but the env variable `SOLANA_KEY_PATH ` is not set. So I updated the response to ask users to create/import their wallets first.